question28/css/news.css

```diff
@@ -0,0 +1,164 @@
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+ol,
+li,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+.container {
+    max-width: 1000px;
+    width: 100%;
+    margin: 30px auto;
+}
+.logout_btn {
+    text-align: right;
+}
+.logout_btn button {
+    padding: 10px 30px;
+    margin-bottom: 30px;
+    background: #eeeeee;
```

Comment:
```suggestion
    background: #eee;
```

Reply:
Thank you for your review! Fixed so please confirm.

fixed commit : https://github.com/csakyo/homework/pull/42/commits/63c2f6372cc0999cac3f56e6be3e015f27b8a01f